README.md

```diff
@@ -0,0 +1,116 @@
+# **ALTERNATE BIT PROTOCOL USING CADMIUM**
+---
+### ORGANINIZATION
```

Comment:
There is a typo error in this line. It would be ORGANIZATION

Reply:
solved